src/backend/tests/it/utils/pocketic.rs

```diff
@@ -50,6 +72,8 @@ pub struct BackendBuilder {
     cycles: u128,
     /// Path to the backend wasm file.
     wasm_path: String,
+    /// Path to the bitcoin canister wasm file.
+    bitcoin_wasm_path: String,
```

Comment:
Each canister should really have its own builder.  If this works, let's run with it for now and refactor later.  There should be a test environment containing canisters.  This here is the builder for the backend canister in that environment and there should be a different canister struct in the environment.  But I realize that we are under time pressure and this can be marked up as tech debt, to be cleaned up later.

Reply:
I followed the same pattern as in the signer canister.

Let's improve it there as well.

Reply:
I created the ticket: https://dfinity.atlassian.net/browse/GIX-3033